distrib/anope/anope2json.py

```diff
@@ -83,6 +84,22 @@ def file_to_objects(infile):
     'SECRET': 's',
 }
 
+# verify that a certfp appears to be a hex-encoded SHA-256 fingerprint;
+# if it's anything else, silently ignore it
+def validate_certfps(certobj):
+    certobj = certobj.split()
+    certfps = []
+    for fingerprint in certobj:
+        try:
+            dec = binascii.unhexlify(fingerprint)
+        except:
+            continue
+        if len(dec) == 32:
+            certfps.append(fingerprint)
+        else:
```

Comment:
You can actually remove this else block; `continue` has no effect here because we're already at the bottom of the loop